src/dummy/toga_dummy/factory.py

```diff
@@ -1,14 +1,16 @@
-from .app import App, MainWindow
-from .color import native_color
+from .app import App, DocumentApp, MainWindow
 from .command import Command
-from .font import Font
+from .documents import Document
+
+from .resources.colors import native_color
+from .resources.fonts import Font
+from .resources.icons import Icon
+from .resources.images import Image
 
 from .widgets.box import Box
 from .widgets.button import Button
 from .widgets.canvas import Canvas
 from .widgets.detailedlist import DetailedList
-from .widgets.icon import Icon
-from .widgets.image import *
 from .widgets.imageview import *
 from .widgets.label import Label
 from .widgets.multilinetextinput import *
```

Comment:
At column 1: (F403) 'from .widgets.multilinetextinput import *' used; unable to detect undefined names